src/software/ai/hl/stp/play/corner_kick_play_test.py

```diff
@@ -11,20 +11,21 @@
 from proto.ssl_gc_common_pb2 import Team
 
 
-def test_corner_kick_play_bottom_left(simulated_test_runner):
-
+def corner_kick_play_test_setup(ball_and_bots_pos, simulated_test_runner):
```

Comment:
pytest is going to try to test everything with "test" in the name, so just call it corner_kick_play_setup